mattermost/mattermost-webapp
# MM-17454 add test for "Scroll to bottom when sending a message"

#### Summary
Add test "M17454 Scroll to bottom when sending a message"

Test steps:

1. login with user-1, go to channel2
2. post a message in channel1 by user-2
3. got to channel1 by user-1, check that separator with a message is visible
4. post a message in channel1 by user-1 check that page scrolled down to that message after posting

#### Ticket Link
Jira ticket [MM-17454](https://mattermost.atlassian.net/browse/MM-17454)
closes https://github.com/mattermost/mattermost-server/issues/11750
